File: 백준/Bronze/3449. 해밍 거리/해밍 거리.c
#include <stdio.h>
#include <string.h>

int main(void) { 
   int a, count;
   char num1[101], num2[101]; 
   scanf("%d", &a);

   for(int i = 0; i < a; i++){
      count = 0;
      scanf("%s", num1);
      scanf("%s", num2);

      for(int j = 0; j < strlen(num1); j++){
         if(num1[j] != num2[j]){
            count++;
         }
      }
      printf("Hamming distance is %d.\n", count);
   }
   
   return 0;
}

File: 백준/Bronze/3449. 해밍 거리/해밍 거리.py
a = int(input())

for _ in range(a):
  n1 = input()
  n2 = input()

  count = 0

  for i in range(len(n1)):
    if n1[i] != n2[i]:
      count += 1

  print(f"Hamming distance is {count}.")